OperationGuide.md

```diff
@@ -0,0 +1,22 @@
+﻿**The use  are divided into the following steps:**
```

Comment:
Use `#` to identify a topic

Reply:
How about _moving_ the documents into a single directory `doc`?

Reply:
OK. A new branch has been pulled to submit this code. This branch will close.